dbms/src/TestUtils/FunctionTestUtils.cpp

```diff
@@ -403,7 +407,7 @@ DataTypePtr getReturnTypeForFunction(
         for (size_t i = 0; i < columns.size(); ++i)
             argument_column_numbers.push_back(i);
         auto columns_with_unique_name = toColumnsWithUniqueName(columns);
-        auto [actions, result_name] = buildFunction(context, func_name, argument_column_numbers, columns_with_unique_name, collator);
+        auto [actions, result_name] = buildFunction(context, func_name, argument_column_numbers, columns_with_unique_name, collator, "");
```

Comment:
Why change this? The last argument should have default value

Reply:
> Why change this? The last argument should have default value

val of buildFunction does not have default value, so I need to explicitly designate it.